source/features/useful-not-found-page.tsx

```diff
@@ -223,6 +243,11 @@ void features.add(import.meta.url, 	{
 		pageDetect.isPRCommit404,
 	],
 	init: onetime(initPRCommit),
+}, {
+	asLongAs: [
```

Comment:
```suggestion
	include: [
```

`include` is the default. `asLongAs` is only used when every condition must be true